### ISSUE TITLE
599 Netscaler specific error: Binding Monitor to GSLB Service

### ISSUE THREAD

**Issue by Priteshkal** [citrix/terraform-provider-citrixadc#67]
Hi,

**ADC Provider**: v12.6

I am trying to bind a "ping" monitor to my gslb service and I keep getting:

```
* module.<app>_gslb.citrixadc_gslbservice.gslb_service: 1 error(s) occurred:

* citrixadc_gslbservice.gslb_service: [ERROR] go-nitro: Failed to create resource of type gslbservice, name=<app>-443-SSL, err=failed: 599 Netscaler specific error ({ "errorcode": 2170, "message": "Monitor not bound to service", "severity": "ERROR" })
```

This is probably due to not having the correct configurations for gslbservice_lbmonitor_binding in the provider.

I tried doing a PUT on the following: **https://\<HOST\>/nitro/v1/config/gslbservice_lbmonitor_binding/\<app\>-443-SSL** 
with this as the body:
**{"gslbservice_lbmonitor_binding":{"monitor_name":"ping","servicename":"\<app\>-443-SSL","weight":1}}** and it worked. 

Can we please fix this in the provider asap?

Thanks

**Comment by Priteshkal** [comment]
Any progress on this issue?

**Comment by giorgos-nikolopoulos** [comment]
No not at the moment.

I am working on some ansible updates.

I will take this up right after I finish the current task.

**Comment by Priteshkal** [comment]
@giorgos-nikolopoulos Can we get this in one of the upcoming releases? Trying to automate my GSLB creation using terraform v0.12.10

**Comment by Priteshkal** [comment]
@giorgos-nikolopoulos Any estimate on when this will be done by?

**Comment by giorgos-nikolopoulos** [comment]
I am working right now on some new resources.
I could probably squeeze this in if it is not an excessive amount of work.

Could you provide me some more context?
What is your resource file like?

How do you trigger this error?

**Comment by giorgos-nikolopoulos** [comment]
Fixed with #74

**Comment by Priteshkal** [comment]
Thank you @giorgos-nikolopoulos! This was exactly what I was looking for.
